doc/tired_support.md

```diff
@@ -0,0 +1,197 @@
+| MIPS         | C     |
+| XTENSA       | C     |
+
+[^1]: will upgrade to **A** after further testing and validation.
+
+## OS / Platform Support
+
+| Platform           | Tier  |
+| ------------------ | ----- |
+| **NuttX**          | **A** |
+| **Ubuntu**         | **A** |
+| Android            | B     |
+| macOS              | B     |
+| Windows            | B     |
+| Zephyr             | B[^2] |
+| AliOS-Things       | C     |
+| Cosmopolitan       | C     |
+| ESP-IDF (FreeRTOS) | C     |
+| FreeBSD            | C     |
+| iOS                | C     |
+| RT-Thread          | C     |
+| RIOT               | C     |
+| VxWorks            | C     |
+
+[^2]: will upgrade to **A** after further testing and validation.
+
+## WebAssembly Proposal Support
+
+> Defaults below reflect the sample build configuration you provided (e.g., `WAMR_BUILD_*` values), not necessarily the release bundle. “Always-on” items are part of WAMR’s baseline.
+
+| WASM Proposal / Extension              | Tier        | Default |
+| -------------------------------------- | ----------- | ------- |
+| **Bulk Memory**                        | A           | **On**  |
+| **Extended Constant Expressions**      | A           | Off     |
+| **Import/Export of Mutable Globals**   | A           | **On**  |
+| **Memory64**                           | A           | Off     |
+| **Multi-value**                        | A           | **On**  |
+| **Non-trapping float-to-int**          | A           | **On**  |
+| **Reference Types**                    | A           | **On**  |
+| **Shared Memory (Threads)**            | A           | Off     |
+| **SIMD (128-bit)**                     | A           | **On**  |
+| **Sign-extension Operators**           | A           | **On**  |
+| **Tail Calls**                         | A           | Off     |
+| GC (Garbage Collection)                | B           | Off     |
+| Stringref                              | B           | Off     |
+| Multi-memory                           | C           | Off     |
+| Legacy Exception Handling              | C           | Off     |
+| Branch Hinting                         | Unsupported |         |
+| Custom Annotation Syntax (text format) | Unsupported |         |
+| Exception Handling (new spec)          | Unsupported |         |
+| JS String Builtins                     | Unsupported |         |
+| Relaxed SIMD                           | Unsupported |         |
+
+# WAMR-Specific Feature Support
+
+> Defaults below mirror your sample build output (e.g., “enabled/disabled” lines) and common WAMR options.
+
+| WAMR Feature                      | Tier | Default |
+| --------------------------------- | ---- | ------- |
+| **AoT (wamrc)**                   | A    | **On**  |
+| **AOT intrinsics**                | A    | **On**  |
+| **Fast Interpreter**              | A    | **Off** |
+| **Interpreter (classic)**         | A    | **On**  |
+| **Libc builtin**                  | A    | **On**  |
```

Comment:
i personally don't recommend the builtin libc for any serious users.
is it still used widely?

Reply:
Widely? No, but still several as known.